test/test_linalg.py

```diff
@@ -1686,59 +1686,64 @@ def test_nuclear_norm_exceptions_old(self, device):
     def test_inverse(self, device, dtype):
         from torch.testing._internal.common_utils import random_fullrank_matrix_distinct_singular_value
 
-        def run_test(matrix, batches, n):
-            matrix_inverse = torch.inverse(matrix)
+        def run_test(torch_inverse, matrix, batches, n):
+            matrix_inverse = torch_inverse(matrix)
 
             # Compare against NumPy output
             # NumPy uses 'gesv' LAPACK routine solving the equation A A_inv = I
```

Comment:
Do you have an opinion on the quality of these two approaches?

Reply:
I think using `getrf` + `getri` is better since that is specifically the intended use of `getri` routine - matrix inversion. While in NumPy it's a kind of a trick to compute inv(A) this way. I don't know the reason behind this implementation.